### DISCUSSION TITLE
Compare Calendar objects

### DISCUSSION THREAD

**Discussion by abe-101** [collective/icalendar#498] [Q&A] [answered]
What would be the correct way to compare calendar objects?

I'm working on a webapp that merges external (link) calendars 
There is a routine task that pulls events from external sources, creates a single calendar object and writes it to a file.
I would only like to write to the file if there are new events
So my question is, after creating a new calendar object from the external sources how can I compare it to the calendar object written to the file?

**Comment by abe-101** [comment]
Here is the file
https://github.com/abe-101/calmerge/blob/3a4fc707649e7af8201c5405d70b59892af28d95/calmerge/calendars/utils.py

**Comment by niccokunzmann** [comment, marked as answer]
I would compare UUID and Recurrence-id. They would identify an event.
